File: PYTHON/Program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
      
    # ATRIBUT-ATRIBUT YANG MAU DIINISIALISASIKAN
    __untuk = ""
    __size = '-'
    __merk = ""

     # CONSTRUCTOR DENGAN PARAMETER UNTUK ME-NEW KAN
    def __init__(self, untuk = "", size = '', merk = ""):
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk

    # Getter
    def get_untuk(self): return self.__untuk
    def get_size(self): return self.__size
    def get_merk(self): return self.__merk

    # Setter
    def set_untuk(self, untuk): self.__untuk = untuk
    def set_size(self, size): self.__size = size
    def set_merk(self, merk): self.__merk = merk

    # DESTRUKTOR
    # def __del__(self):
    #     print("asdsr")

File: PHP/Program/Baju.php
<?php

require ('Aksesoris.php');

class Baju extends Aksesoris
{
    // Atribut-atribut yang akan dipakai pada main.php
    private $untuk = "";
    private $size = '-';
    private $merk = "";

    // CONSTRUCTOR dengan parameter
    public function __construct($untuk = "", $size = '', $merk = "")
    {
        parent::__construct();
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    // GETTER SETTER

    // Setter ID
    public function set_untuk($untuk)
    {
        $this->untuk = $untuk;
    }

    // Getter ID
    public function get_untuk()
    {
        return $this->untuk;
    }

    // Setter nama produk
    public function set_size($size)
    {
        $this->size = $size;
    }

    // Getter nama produk
    public function get_size()
    {
        return $this->size;
    }

    // Setter stok produk
    public function set_merk($merk)
    {
        $this->merk = $merk;
    }

    // Getter stok produk
    public function get_merk()
    {
        return $this->merk;
    }

    // DESTRUKTOR
    public function __destruct()
    {
        
    }
}
